elastic/kibana
# [Logs onboarding] Explore logs button should have different states

Closes https://github.com/elastic/kibana/issues/166009.

When system integration is installed successfully we should be able to redirect the user to logs explorer with the `System integration` selected

https://github.com/elastic/kibana/assets/1313018/7a1d7457-00e8-4399-9332-3ea7af2b0145

When system integration installation failed we should be able to redirect the user to logs explorer with `All logs` selected

https://github.com/elastic/kibana/assets/1313018/e13f69ab-adb4-460c-84cf-d9971ce56b9f

When system integration is being installed, Explore Logs button should be disabled

https://github.com/elastic/kibana/assets/1313018/5dd04859-e75b-49ed-9f47-5ad2120eb9d8

### How to test
1. Log in as `viewer` user
2. Enter [system logs](https://deploy-kibana-pr167409.kb.us-west2.gcp.elastic-cloud.com/app/observabilityOnboarding/systemLogs) onboarding flow 
3. Verify System integration was not installed
4. Click on `Explore Logs`
5. Verify you are being redirected to Log explorer with `All logs` being selected
6. Logout and log in again as `admin` user
7. Enter [system logs](https://deploy-kibana-pr167409.kb.us-west2.gcp.elastic-cloud.com/app/observabilityOnboarding/systemLogs) onboarding flow 
8. Verify system integration was installed successfully
9. Click on `Explore Logs`
10. Verify you are being redirected to Log explorer with `[system]` being selected
